front/app/containers/EmailChange/UpdateEmailForm.tsx

```diff
@@ -67,7 +68,7 @@ const UpdateEmailForm = ({
           });
       } else {
         // Otherwise, update the user's email
-        await updateUser(user.id, { ...formValues });
+        await updateUser({ userId: user.id, ...formValues });
         setUpdateSuccessful(true);
```

Comment:
In this case - is there any difference between using the `await` versus putting the `setUpdateSuccessful(true);` in the onSuccess instead? :thinking:

Reply:
I think the form validation here relies on this being a promise and I didn't want to refactor it at this time.